src/ui.rs

```diff
@@ -261,6 +261,14 @@ impl App {
                 self.previous();
                 AppKeyAction::Ok
             }
+            Char('n') => {
+                self.next();
+                AppKeyAction::Ok
+            }
+            Char('p') => {
+                self.previous();
+                AppKeyAction::Ok
+            }
```

Comment:
```suggestion
            Char('j') | Char('n') => {
                self.next();
                AppKeyAction::Ok
            }
            Char('k') | Char('p') => {
                self.previous();
                AppKeyAction::Ok
            }
```

Hey 👋 You can do like this in Rust

Reply:
Thanks!

Reply:
```rust
            Char('j') | Char('n') => {
                self.next();
                AppKeyAction::Ok
            }
            Char('k') | Char('p') => {
                self.previous();
                AppKeyAction::Ok
            }
```
The above gave an issue during compilation.

It probably should be:
```rust
            Char('j' | 'n') => {
                self.next();
                AppKeyAction::Ok
            }
            Char('k' | 'p') => {
                self.previous();
                AppKeyAction::Ok
            }
 ```

Reply:
Perfect! Thanks for the contribution 🙏